CMakeLists.txt

```diff
@@ -84,7 +84,15 @@ endif()
 if(NOT TARGET FARGPARSE::fargparse)
   find_package(FARGPARSE QUIET)
 endif()
-if(NOT TARGET YAFYAML::yafyaml)
+
+
+option(USE_EXTDATA2G "Use ExtData2G" ON)
+if(USE_EXTDATA2G)
+  set (EXTDATA2G_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData2G Target")
+  find_package(YAFYAML REQUIRED)
+  message (STATUS "Building with ExtData2G")
+else()
+  set (EXTDATA2G_TARGET "" CACHE STRING "ExtData2G Target")
   find_package(YAFYAML QUIET)
 endif()
```

Comment:
```suggestion
option(USE_EXTDATA2G "Use ExtData2G" ON)
if(USE_EXTDATA2G)
  set (EXTDATA2G_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData2G Target")
  if (NOT TARGET YAFYAML::yafyaml)
    find_package(YAFYAML REQUIRED)
  endif ()
  message (STATUS "Building with ExtData2G")
else()
  set (EXTDATA2G_TARGET "" CACHE STRING "ExtData2G Target")
endif()
```

Reply:
This just removes find_package for the extdata1 case (not needed) and adds back the `IF NOT TARGET` for GCHP folks.